tests/shared/src/test/scala-2.13/scala/meta/tests/tokenizers/TokensPositionSuite.scala

```diff
@@ -158,30 +158,37 @@ class TokensPositionSuite extends BasePositionSuite(dialects.Scala213) {
   )
   checkPositions[Stat](
     "(f)((((a))))",
-    """|Term.ArgClause ((((a))))
+    """|Term.Name (f)
```

Comment:
I don't think name should contain `()`. Might be problematic if anyone wants to get the identifier position.

Reply:
then let me try once again. i had a really, really hard time getting `scalafmt` to work when parens became part of the parent. perhaps we just need to make a few changes in the formatter not to rely on these.

Reply:
question: in `(a).b`, which tree should own the parens? (i'd prefer to include in `a` as that's what they are designed to encompass)

Reply:
I would say `(a).b` since it's not really part of anything else, but the select. Was it previously containing (a) for Term.Name? If that is the case we can revert if it's causing issues

Reply:
let me try once again. i spent a week trying to fix all the scalafmt failures with the current scalameta, but couldn't do it. generally, commentary around the formatter codebase talks about scalameta helping the formatter rather than requiring massive special-case processing, so since i (possibly, misguidedly but with same logic as yours) removed these positions in #2941, this PR was an attempt to restore some of them.

Reply:
I am fine with reverting, especially if it was that way before.